answers.md

```diff
@@ -1 +1,220 @@
 Your answers to the questions go here.
+# About Datadog
+Datadog is a data monitoring service for cloud-scale applications, bringing together data from servers, databases, tools, and services to present a unified view of an entire stack. These capabilities are provided on a SaaS-based data analytics platform. [Wiki](https://en.wikipedia.org/wiki/Datadog)
+
+### Features
+* Observability - From infrastructure to apps, in any environment
+* Dashboards - Use instant, real-time boards or build your own
+* Infrastructure - From overview to deep details, fast
+* Analytics - Custom app metrics or business KPIs
+* Collaboration - Share data, discuss in context, solve issues quickly
+* Alerts - Avoid alert fatigue with smart, actionable alerts
+* API - Love infrastructure as code? You'll love Datadog's API
+* Machine Learning - Automatically detect outliers and temporal anomalies
+* APM - Monitor, optimize, and troubleshoot app performance
+
+<a href="http://www.flickr.com/photos/alq666/10125225186/" title="The view from our roofdeck">
+<img src="http://farm6.staticflickr.com/5497/10125225186_825bfdb929.jpg" width="500" height="332" alt="_DSC4652"></a>
+
+I am here to apply for the support engineer at [Datadog](http://datadog.com) Sydney.
+
+
+# The Challenge
+
+## Questions
+
+### Level 0 (optional) - Setup an Ubuntu VM
+
+* While it is not required, we recommend that you spin up a fresh linux VM via Vagrant or other tools so that you don't run into any OS or dependency issues. [Here are instructions for setting up a Vagrant Ubuntu 12.04 VM.](https://www.vagrantup.com/docs/getting-started/)
+>Answer: I am using macOS Sierra Version 10.12.6 for this Challenge.
+
+
+### Level 1 - Collecting your Data
+
+* Sign up for Datadog (use "Datadog Recruiting Candidate" in the "Company" field), get the Agent reporting metrics from your local machine.
+
+>Answer: [Sign up here](https://www.datadoghq.com/#), get a datadog account for free for 14 days.
+
+>login, click on [_integration-Agent_](https://app.datadoghq.com/account/settings#agent/mac) in DataDog on the left column and follow the installation instructions for Mac OS X to install the Agent.
+
+> The datadog can be installed on OS X as easily as:
+```
+DD_API_KEY=63ab065b2982aed65fff538ba18a93ba bash -c "$(curl -L https://raw.githubusercontent.com/DataDog/dd-agent/master/packaging/osx/install.sh)"
+```
+<img src="https://github.com/jinmei612/datadog_screenshots/blob/master/upload/Agent%20instalation%20and%20config.png" />
+
+* Bonus question: In your own words, what is the Agent?
+
+>Answer: The Datadog agent is a full stack data monitoring platform/software which brings data from servers, databases, tools, and services to the Datadog web interface.
+
```

Comment:
ANSWER: The Datadog agent collects the data from the devices (servers, dbs, tools, containers) to the Datadog Web UI.